### ISSUE TITLE
CentOS 8.2 - invalid arithmetic operator on line 127 and 177 in hyperion-prepare.sh

### ISSUE THREAD

**Issue by N3Cr0N** [wrljet/hercules-helper#1]
Hi,
i have an issue with your "hyperion-prepare.sh" Script on line 127 and 177 in CentOS.

```
[root@centos ~]# ./hercules-helper/hyperion-prepare.sh
We have a CentOS system
VERSION_MAJOR : 8.2
./hercules-helper/hyperion-prepare.sh: line 127: [[: 8.2: syntax error: invalid arithmetic operator (error token is ".2")
We have a CentOS system
VERSION_MAJOR = 8.2
./hercules-helper/hyperion-prepare.sh: line 177: [[: 8.2: syntax error: invalid arithmetic operator (error token is ".2")
Done!

```

My System:
 
```
[root@centos ~]# uname --all
Linux centos 4.18.0-193.28.1.el8_2.x86_64 #1 SMP Thu Oct 22 00:20:22 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

kind regards

**Comment by wrljet** [comment]
I'll look into that for you.

**Comment by wrljet** [comment]
This should be fixed with commit 51c9e7e.
Tested on CentOS 7.8 and 8.2.

Must be run as a normal user, not root.

**Comment by N3Cr0N** [comment]
This bug is fixed, but i have 2 another Bugs under CentOS 
for one i open a pull request https://github.com/wrljet/hercules-helper/pull/5/commits/d20a65fa18f00f87a059ba3632adb64c225d9a19
